File: Examples/DotNET/CSharp/Paragraph/GetListOfParagraphs.cs
using System;
using Com.Aspose.Words.Api;
using Com.Aspose.Words.Model;
using Com.Aspose.Storage.Api;

namespace Paragraph
{
    class GetListOfParagraphs
    {
        public static void Run()
        {
            // ExStart:1
            WordsApi wordsApi = new WordsApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            String fileName = "SampleWordDocument.docx";
            String storage = null;
            String folder = null;

            try
            {
                // Upload the file
                storageApi.PutCreate(fileName, "", storage, System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName));
                // Invoke Aspose.Words Cloud SDK API to get list of paragraphs from word document
                ParagraphLinkCollectionResponse apiResponse = wordsApi.GetDocumentParagraphs(fileName, storage, folder);

                if (apiResponse != null && apiResponse.Status.Equals("OK"))
                {
                    ParagraphLink docProperty = apiResponse.Paragraphs.ParagraphLinkList[0];

                    // Display document paragraphs info
                    foreach (ParagraphLink docParagraphLink in apiResponse.Paragraphs.ParagraphLinkList)
                    {
                        Console.WriteLine("Link : " + docParagraphLink.link);
                        Console.WriteLine("Text : " + docParagraphLink.Text);
                    }
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}

File: Examples/Node.js/Paragraph/GetListOfParagraphs.js
// ExStart:1
var fs = require('fs');
var assert = require('assert');
var StorageApi = require('asposestoragecloud');
var WordsApi = require('asposewordscloud');
var configProps = require('../Config/config.json');
var data_path = '../../../Data/';

var AppSID = configProps.app_sid;
var AppKey = configProps.api_key;
var config = {'appSid':AppSID,'apiKey':AppKey , 'debug' : true};

// Instantiate Aspose Storage API SDK
var storageApi = new StorageApi(config);
// Instantiate Aspose Words API SDK
var wordsApi = new WordsApi(config);

// Set input file name
var name = "SampleWordDocument.docx";
try {
// Upload source file to aspose cloud storage
storageApi.PutCreate(name, versionId=null, storage=null, file= data_path + name , function(responseMessage) {

	assert.equal(responseMessage.status, 'OK');

	// Invoke Aspose.Words Cloud SDK API to get list of paragraphs from a word document
	wordsApi.GetDocumentParagraphs(name, null, null, function(responseMessage) {
			assert.equal(responseMessage.status, 'OK');
			
			// Display document paragraphs info
			responseMessage.body.Paragraphs.ParagraphLinkList.forEach(function(docParagraphLink) {
				console.log("Link Href: " + docParagraphLink.link.Href);
				console.log("Text : " + docParagraphLink.Text);
				});
			});
	});

}catch (e) {
  console.log("exception in example");
  console.log(e);
}
//ExEnd:1